ui-components/src/blocks/AppStatusItem.tsx

```diff
@@ -1,34 +1,20 @@
 import React from "react"
 import "./AppStatusItem.scss"
 import moment from "moment"
-import { MultiLineAddress } from "../helpers/address"
-import { t } from "../helpers/translator"
 import { Application, Listing } from "@bloom-housing/backend-core/types"
 import { LocalizedLink } from "../actions/LocalizedLink"
+import { t } from "@bloom-housing/ui-components"
 
-// TODO status and lotteryNumber should be loaded from Application
 interface AppStatusItemProps {
   application: Application
   listing: Listing
-  status: string
-  setDeletingApplication: (application: Application) => void
-  lotteryNumber?: string
```

Comment:
I'm not familiar with the changes to remove this AppStatusItem functionality. Can you let me know why this changed?

Reply:
Sure! So the designs I had didn't have anything in them to delete applications so I removed that piece, and then there was an older TODO on line 9 "// TODO status and lotteryNumber should be loaded from Application"